test-suite/exonum-py-tests/exonum_tests/api.py

```diff
@@ -106,6 +113,7 @@ def test_get_n_latest_blocks(self):
             host, public_port, private_port = self.network.api_address(validator_id)
             client = ExonumClient(host, public_port, private_port)
             blocks_response = client.get_blocks(count=number_of_blocks, latest=latest)
+            self.assertEqual(len(blocks_response.json()['blocks']), latest+1)
```

Comment:
Shouldn't it be `self.assertEqual(len(blocks_response.json()['blocks']), number_of_blocks)`?